src/main/java/org/veupathdb/service/eda/ds/plugin/pass/MapPlugin.java

```diff
@@ -142,8 +173,11 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
         .getBytes()
       );
     }
-    // close array and enclosing object
-    out.write("]}".getBytes());
+    // add config and close
+    // bc the only var we take is geoVar these two values are the same
```

Comment:
I missed this - then how does it work when we have the x-axis variable? We want hair color as x axis on pies, say. Then do we not have a geo variable and hair color var?

Reply:
multiple requests.. this gives the location of map markers and entity counts only. to get data for the pies there is a second request to the pieplot endpoint where the facet variable is the geoVar. i mean really to call these axes vars here hardly makes sense. im considering not reusing PlotConfig and just having the and explicit single 'completeCasesGeoVar'